### ISSUE TITLE
Can't open new dashboard

### ISSUE THREAD

**Issue by Aventura5** [rakanalh/emacs-dashboard#33]
I cant figure out how to open a new dashboard if I closed the one that opens with Emacs. I've tried opening a new buffer and running `M-x dashboard-mode` which has no visible effect, and then I saw `dashboard-insert-startupify-lists` and tried that which also did nothing. I'm not sure if it has anything to do but if i hit `g` (reload) on the dashboard that opens with Emacs nothing happens and it says that `dashboard-insert-startupify-lists` is bound to `g`.

**Comment by rakanalh** [comment]
Try using C-x b and switch to the `*Dashboard*` buffer. If no such buffer exists, then use M-x `dashboard-insert-startupify-lists` and then switch to that buffer.

**Comment by Aventura5** [comment]
Oh, thanks. I had tried `dashboard-mode` and *then* `dashboard-insert-startupify-lists`. Refresh with `g` still doesn't work though, if I find a file and then hit `g` the recents list stays the same.

**Comment by vitaminless** [comment]
In the mode help g is bound to `dashboard-insert-startupify-lists` and it seems to me that it should perform a refresh. I tried to add my org agenda list to the dashboard and discovered that dashboard never seems to update while emacs running.

So may there are to separate bugs/features, one would be a command to invoke the dashboard and switch to it, it would be useful if I killed the buffer and more useable than the way its today.
The second would be a way to proper perform updates at runtime, may there should be a command and a function.

Anyway great work and I'm looking forward to proper integrate org-mode.
